doc/advanced-configuration/database-backends-configuration.md

```diff
@@ -206,15 +206,16 @@ cat azuresql.sql | tr -d '\r' | tr '\n' ' ' | sed 's/GO/\n/g' |
 isql mongoose-mssql username password -b
 ```
 
-The final step is to configure ``mongooseim.cfg`` appropriately.
-Configure the database section as follows:
+The final step is to configure ``mongooseim.toml`` appropriately.
+Configure the `outgoing_pools.rdbms` section as follows:
 
-```erlang
-{rdbms_server_type, mssql}.
-{outgoing_pools, [
- {rdbms, global, default, [{workers, 5}],
-  [{server, "DSN=mongoose-mssql;UID=username;PWD=password"}]}
-]}.
+```toml
+[outgoing_pools.rdbms.default]
```

Comment:
`rdbms_server_type = "mssql"` is still required I think.

Reply:
Yes, put it in the `general` section